src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -1109,7 +1110,10 @@ export class Workbench implements IPartService {
 			this.setPanelHidden(true, true).done(undefined, errors.onUnexpectedError);
 			this.setSideBarHidden(true, true).done(undefined, errors.onUnexpectedError);
 
-			this.setActivityBarHidden(true, true);
+			this.setActivityBarHidden(false, false);
```

Comment:
This line is not needed. `this.setActivityBarHidden(false, false);`
If the setting is not `true` we simply should not do anything